Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -362,7 +362,17 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
         // Compute the map of known versions.
         //
         // FIXME: Move this utility to a more stable location.
-        let knownVersions = Git.convertTagsToVersionMap(repository.tags)
+        let knownVersionsWithDuplicates = Git.convertTagsToVersionMap(repository.tags)
+        
+        let knownVersions = knownVersionsWithDuplicates.mapValues { tags -> String in
```

Comment:
If the closure doesn't have any side effects, avoid using trailing closure syntax:
http://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/

Reply:
Oh wow, I didn't know this. Thank you!